app/controllers/api/v1/base_controller.rb

```diff
@@ -20,7 +20,7 @@ class Api::V1::BaseController < ApplicationController
     render json: { error: ex.message }, status: :unauthorized
   end
 
-  rescue_from_unless_local ActiveRecord::RecordNotFound, send_to_sentry: false do |ex|
+  rescue_from_unless_local ActiveRecord::RecordNotFound, send_to_sentry: true do |ex|
```

Comment:
hmm, this means it's going to send all not founds to sentry - I guess this is good to start with but it could get quite noisy.